fpdf/fpdf.py

```diff
@@ -148,6 +148,7 @@ def __init__(self, orientation="P", unit="mm", format="A4"):
         self.links = {}  # array of internal links
         self.in_footer = 0  # flag set when processing footer
         self.lasth = 0  # height of last cell printed
+        self.current_font = {}  # current font
```

Comment:
Seems unused

Reply:
@Lucas-C It's used for example here:

`self.current_font = self.fonts[fontkey]`

and as well in many other places throughout the `fpdf.py` file. I just initialized it.